### ISSUE TITLE
Marker/dot missing

### ISSUE THREAD

**Issue by nullket** [pR0Ps/grafana-trackmap-panel#63]
Hi,

i saw the closed issue concerning the missing marker/dot in older versions: https://github.com/pR0Ps/grafana-trackmap-panel/issues/57, https://github.com/pR0Ps/grafana-trackmap-panel/issues/50 

I have the plugin in version `2.1.2` installed in grafana `8.0.3` and it works fine - except the blue dot for the last position is missing. I never used this plugin before, so I don't know if it is a grafana version issue, influx 2.0 query issue or something else. Please let me know what kind of information or data could be used to solve this issue. 

For record, this my influx 2.0 flux query (fields in the db are named "lat" and "long" and if filter an additional "status_flag"

```flux
Query A
from(bucket: "parser")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r["_measurement"] == "duro")
  |> filter(fn: (r) => r["_field"] == "lat")
  |> filter(fn: (r) => r["status_flag"] == "4")
  |> aggregateWindow(every: v.windowPeriod, fn: last, createEmpty: false)
  |> set(key: "_field", value: "latitude")
  |> yield(name: "last")

Query B
from(bucket: "maas-vm-parser")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r["_measurement"] == "duro")
  |> filter(fn: (r) => r["_field"] == "lon")
  |> filter(fn: (r) => r["status_flag"] == "4")
  |> aggregateWindow(every: v.windowPeriod, fn: last, createEmpty: false)
  |> set(key: "_field", value: "longitude")
  |> yield(name: "last")

```

**Comment by pR0Ps** [comment]
If you see datapoints displayed on the map then your query is working properly.

It's probably an issue with the plugin not properly receiving "on hover" events from Grafana anymore due to a change in a recent version.

**Comment by nullket** [comment]
Yes, the data is displayed (just without the dot). 

I just figured out that I can get the dot to be displayed when I hover over **other** panels which are of the type "Graphs (old)" - whereas the newer "Time Series" panel does not trigger such an behavior.

**Comment by Bart-1992** [comment]
Will this issue (ever) be solved? Or is there a solution on how to solve this?

**Comment by pR0Ps** [comment]
Fixed by #77 in v2.1.3
